<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillsDialog</class>
 <widget class="QDialog" name="BillsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="bills_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>425</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="balance_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>余额支付</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wechat_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>微信支付</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_lb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>合计：0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="zfb_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>支付宝支付</string>
    </property>
   </widget>
   <widget class="HistoryListWidget" name="pay_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title_lb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
padding: 8px 12px;</string>
    </property>
    <property name="text">
     <string>待缴费账单明细</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="qr_lb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="status_lb">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>status_lb</zorder>
   <zorder>balance_btn</zorder>
   <zorder>wechat_btn</zorder>
   <zorder>total_lb</zorder>
   <zorder>zfb_btn</zorder>
   <zorder>pay_widget</zorder>
   <zorder>title_lb</zorder>
   <zorder>close_btn</zorder>
   <zorder>qr_lb</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryListWidget</class>
   <extends>QListWidget</extends>
   <header>historylistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
